app/src/forms/agriseafoodopscreening/emailService.js

```diff
@@ -0,0 +1,169 @@
+const chesService = require('../../components/chesService');
```

Comment:
Hopefully can commonize this (among other), as it's the same functionality for multiple forms